XRTK-Core/Packages/com.xrtk.core/Interfaces/CameraSystem/IMixedRealityCameraSystem.cs

```diff
@@ -10,28 +11,31 @@ namespace XRTK.Interfaces.CameraSystem
     public interface IMixedRealityCameraSystem : IMixedRealityService
     {
         /// <summary>
-        /// Is the current camera displaying on an Opaque (AR) device or a VR / immersive device
+        /// The list of <see cref="IMixedRealityCameraDataProvider"/>s registered and running with the system.
         /// </summary>
-        bool IsOpaque { get; }
+        IReadOnlyCollection<IMixedRealityCameraDataProvider> CameraDataProviders { get; }
 
         /// <summary>
-        /// Is the current camera displaying on a traditional 2d screen or a stereoscopic display?
+        /// The reference to the <see cref="IMixedRealityCameraRig"/> attached to the Main Camera (typically this is the player's camera).
         /// </summary>
-        bool IsStereoscopic { get; }
+        IMixedRealityCameraRig MainCameraRig { get; }
```

Comment:
I don't agree with simply renaming this as it causes so much downstream change for no benefit.
Why change the name?  This is a major breaking change

Reply:
the entire `IMixedRealityCameraSystem` interface changed. I also wanted to make a distinction here. It's possible we may have multiple cameras that have different rig requirements in the future.